client/components/codeEditor.jsx

```diff
@@ -25,26 +24,24 @@ export default function CodeEditor(props) {
     return () => clearTimeout(timeout);
   }, [html, css, javascript]);
 
-  function addtoDatabase() {
-    const newObject = { html, css, javascript, title, imageUrl, description };
-    fetch('/api/code', { method: 'POST', body: JSON.stringify(newObject), headers: { 'Content-Type': 'application/json' } });
-
-  }
   function handleTitle(event) {
     setTitle(event.target.value);
   }
   function handleImage(event) {
     setImageUrl(event.target.value);
-
   }
   function handleDescription(event) {
     setDescription(event.target.value);
   }
-
+  function handleSubmit() {
+    const newObject = { html, css, javascript, title, imageUrl, description };
+    fetch('/api/code', { method: 'POST', body: JSON.stringify(newObject), headers: { 'Content-Type': 'application/json' } });
+    window.location.hash = '#';
```

Comment:
I recommend waiting until the fetch completes before changing location hash because the browser might change to a new page before the data is saved.

```js
fetch('/api/code', { method: 'POST', body: JSON.stringify(newObject), headers: { 'Content-Type': 'application/json' } })
  .then(() => {
    window.location.hash = '#';

  });
```